web/app/onboard/onboard.component.ts

```diff
@@ -44,6 +47,12 @@ export class OnboardComponent {
     });
   }
 
+  submitOnboarding(): void {
+    if (this.form.valid) {
+      console.log('Onboarding is being submitted');
```

Comment:
Do you want this to be merged or was that for testing only?

Reply:
i think it's OK to merge for now since it's still in development

Reply:
Alright